docs/source/conf.py

```diff
@@ -0,0 +1,25 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "uharfbuzz"
+copyright = "2024, Adrien Tétar"
+author = "Adrien Tétar"
```

Comment:
May be we can use something more generic, like “HarfBuzz team” or something? Adrien hasn’t been involved with this project for a while now.

Reply:
I've now changed the `author` to `HarfBuzz Contributors` to adopt this :+1:

Reply:
(as I am a lowly first-time contributor I've left the copyright notice as-is, but have checked "Allow edits by maintainers" on this PR in case any maintainer wishes to set it)